src/pages/landingPage/navbar/NavbarComponent.tsx

```diff
@@ -0,0 +1,15 @@
+import LeftSection from "./LeftSection";
+import RightSection from "./RightSection";
+
+const NavbarComponent = () => {
+  return (
+    <div className="w-11/12 mx-7">
```

Comment:
Why use w-11/12 here for the whole navbar component?

Reply:
Because when i trying to set margin to the navbar, it doesn't work. So i set the width to 11/12. Sorry, i don't know how to solve this problem.